packages/styles/src/modules/pagination.css

```diff
@@ -0,0 +1,191 @@
+/**
+ * Used to split large content into several pages, allowing users to navigate between them instead of displaying all information on a single page.
+ * @name sd-pagination
+ * @status stable
+ * @since 1.0
+ * @variant { simple } sd-pagination--... The pagination format.
+ * @boolean sd-pagination--inverted Inverts the pagination style.
+ */
+
+.sd-pagination {
+  > :not(ul) {
+    position: absolute;
+    overflow: hidden;
+    height: 1px;
+    width: 1px;
+  }
+
+  ul {
+    @apply flex items-center gap-2;
+
+    li a {
+      @apply text-primary flex items-center justify-center;
+
+      &:focus-visible {
+        @apply outline outline-2 outline-offset-2 outline-primary;
+      }
+    }
+
+    /* Previous and Next */
+    li:first-child,
+    li:last-child {
+      @apply w-12 h-12;
+
+      a {
+        @apply w-full h-full;
+      }
+
+      sd-icon {
+        @apply w-6 h-6;
+      }
+    }
+
+    /* Numbers */
+    li {
+      @apply flex items-center justify-center text-center w-8 h-8;
+
+      &:has(a[data-page]) {
```

Comment:
Why do we need `data-page`? Can we get rid of it? Should we have a look together?